launch_xml/test/launch_xml/test_boolean_substitution.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 Open Source Robotics Foundation, Inc.
```

Comment:
@kenji-miyake meta: some tests in `launch_yaml` would be nice too.

Reply:
Okay, I'll add tests!

Reply:
Added in 8e27b4602a1f4f91a4b12f77b364aab77cad5c01.